src/oc/nrnran123.h

```diff
@@ -56,10 +56,11 @@ extern double nrnran123_dblpick(nrnran123_State*); /* uniform open interval (0,1
 /* nrnran123_dblpick minimum value is 2.3283064e-10 and max value is 1-min */
 
 /* nrnran123_negexp min value is 2.3283064e-10, max is 22.18071 */
+extern double nrnran123_normal(nrnran123_State*); /* mean 0.0, std 1.0 */
```

Comment:
I don't know how this ever got outside the extern "C" section but I think the other three should go inside as well.

Reply:
It was from my testing, trying to see exactly what is used in the models. It will help us for when we will fully go to cpp.

Reply:
The other 3 should go back in if we expect them to be used in models. No model known so far (to me at least) does.

Reply:
I think we can leave them where they are for now.